src/psyclone/psyir/tools/dependency_tools.py

```diff
@@ -153,7 +153,7 @@ def is_array_parallelisable(self, loop_variable, var_info):
         :return: whether the variable can be used in parallel.
```

Comment:
I know you've not touched it here but I'm wondering whether we can make the name of this routine a bit clearer. Perhaps `array_access_parallelisable` as obviously we are not trying to parallelise an array.